### DISCUSSION TITLE
Editing Viewports

### DISCUSSION THREAD

**Discussion by tompropst** [tbabej/taskwiki#403] [Q&A] [answered]
I would like to be able to edit a viewport by navigating to a specific location before entering edit mode. Currently, I only see the heading text until I enter edit mode but then I cannot navigate the line. If I exit edit mode, I again only see the heading text. Is this something unique to my environment or the same for everyone?

I'm using Markdown syntax.

**Comment by liskin** [comment, marked as answer]
Try setting `let g:taskwiki_disable_concealcursor = 1`, that should do the trick (and see the docs for details about what it does).
